src/navigation.tpl

```diff
@@ -18,6 +18,7 @@
 				<li><a href="installconfig-server-iso.html">Install from Ubuntu Server ISO</a></li>
 				<li><a href="installconfig-package-install.html">Install from packages</a></li>
 				<li><a href="installconfig-lxd-install.html">Install into LXD</a></li>
+				<li><a href="installconfig-upgrade.html">Upgrade from MAAS 1.9</a></li>
```

Comment:
I think the link and the visible text can be improved by being explicit that this involves going from 1.9 to 2.0.

`<li><a href="installconfig-upgrade-1.9-to-2.0.html">Upgrade from 1.9 to 2.0</a></li>`